ios/RNDateTimePicker.xcodeproj/project.pbxproj

```diff
@@ -9,6 +9,7 @@
 /* Begin PBXBuildFile section */
 		1415EF09223110200027D3C6 /* RNDateTimePickerManager.m in Sources */ = {isa = PBXBuildFile; fileRef = 1415EF07223110200027D3C6 /* RNDateTimePickerManager.m */; };
 		B3E7B58A1CC2AC0600A0062D /* RNDateTimePicker.m in Sources */ = {isa = PBXBuildFile; fileRef = B3E7B5891CC2AC0600A0062D /* RNDateTimePicker.m */; };
+		F889C30D28D05FB9002BF499 /* RNDateTimePickerShadowView.m in Sources */ = {isa = PBXBuildFile; fileRef = F889C30C28D05FB9002BF499 /* RNDateTimePickerShadowView.m */; };
```

Comment:
Is it necessary to keep this `pbxproj` around? I believe we should rather edit and add files via the example project and get rid of the `pbxproj` file altogether.